app/src/androidTest/java/org/xwiki/android/sync/utils/SharedPrefsUtilsTest.java

```diff
@@ -24,7 +27,7 @@
 
     @Before
     public void setUp() throws Exception {
-        mContext = AppContext.getInstance().getApplicationContext();
+        mContext = getAppContextInstance().getApplicationContext();
```

Comment:
Why do you use function instead of property here?

Reply:
Menas to directly use the instance from AppContext. Like AppContext.instance ?

Reply:
Almost. I think, it must be something like `appContext` property